COMOPT-2359 Document Private Catalog Views and Restricted Access Keys for GraphQL (#354)
AdobeDocs/commerce-services: src/pages/optimizer/merchandising-services/limitations.md

  For details about these limits, see [Boundaries and Limits](https://experienceleague.adobe.com/en/docs/commerce/optimizer/boundaries-limits) in the *Adobe Commerce Optimizer Guide*.

## Private catalog view limits

> - **Restricted Access Keys per catalog view**: Maximum of 3

Reviewer: Global: You're over-capitalizing restricted access keys and private catalog view. I checked your parallel PR on ExL, and you're not doing it there.

> - **Restricted Access Keys per catalog view**: Maximum of 3

Author: ```suggestion
- **Restricted access keys per catalog view**: Maximum of 3
```

> - **Restricted Access Keys per environment**: Maximum of 100

Reviewer 2: I'm not sure this statement is enforced

Author: Leaving in documentation per [Naomi's direction](https://magento.slack.com/archives/C0BMDUN0EA0/p1786041629706059).

```
1. let's document it, but we don't/won't auto-enforce limit currently .
2. they would need to build a Paas customization that generates the key pairs (for now) .
```

> - **Restricted Access Keys per environment**: Maximum of 100

Author: ```suggestion
- **Restricted access keys per environment**: Maximum of 100
```

- **RSA key size**: PEM-encoded RSA public keys must be between 2048 and 8192 bits

> - **Key propagation on revocation**: Removing or deleting a Restricted Access Key may take a short time (observed up to approximately 5 minutes) to take effect due to caching, after which requests using tokens signed by that key are denied

Author: ```suggestion
- **Key propagation on revocation**: Removing or deleting a restricted access key may take a short time (observed up to approximately 5 minutes) to take effect due to caching, after which requests using tokens signed by that key are denied
```

> - **Key propagation on revocation**: Removing or deleting a restricted access key may take a short time (observed up to approximately 5 minutes) to take effect due to caching, after which requests using tokens signed by that key are denied

Author: ```suggestion
- **Key propagation on revocation**: A recently rotated or deleted restricted access key may continue to grant access for a short time (typically up to 5 minutes) due to caching.
```
